.github/workflows/pypi.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish to PyPI
+
+on:
+  push:
+    tags: [ 'v*' ]
+
+env:
+  DEFAULT_PYTHON: 3.9
+
+jobs:
+  publish:
+    name: Publish to PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Nitpick: it's recommended that every step has a name set for better maintainability (but this changes nothing functionally so it's up to you)